include/boost/geometry/algorithms/detail/envelope/initialize.hpp

```diff
@@ -0,0 +1,85 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2015, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Menelaos Karavelas, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
```

Comment:
I have updated all `expand` and `envelope` related files to use the latest wording.

In any case we should go over all files and change at least the two-line versions and eventually all of them to the latest version.